packages/devextreme/testing/testcafe/helpers/accessibility/testAccessibility.ts

```diff
@@ -0,0 +1,91 @@
+import createWidget, { WidgetName } from '../createWidget';
+import { a11yCheck, A11yCheckOptions, ElementContext } from './accessibilityUtils';
+import { isMaterialBased } from '../themeUtils';
+
+export interface Options {
+  [key: string]: any[];
+}
```

Comment:
let's write proper typing for this

Reply:
We don't know how many options the developer will pass and what kind of options they will be. They may differ from component to component. But they should be in array format for convenience and in order not to complicate the algorithm of configuration generation.

Reply:
https://github.com/marker-dao/DevExtreme/pull/2/files